app/plugin.go

```diff
@@ -146,6 +147,17 @@ func (a *App) InitPlugins(pluginDir, webappPluginDir string) {
 	})
 
 	a.SyncPluginsActiveState()
+
+	// CleanUp expired plugin keys every minute
+	go func() {
+		ticker := time.NewTicker(60 * time.Second)
+		defer ticker.Stop()
+
+		for {
+			<-ticker.C
+			a.DeleteAllExpiredPluginKeys()
+		}
+	}()
```

Comment:
@crspeller thoughts on doing this here in a goroutine vs. as a job?

Reply:
I think this makes sense as a job rather than a loose goroutine. Even though it's really simple.
@DSchalla Can you move this over to using a job? You can see an example in `migrations/scheduler.go` and `migrations/worker.go`.

Reply:
Sure, sounds good! I'll update the PR.